pbcommand/models/report.py

```diff
@@ -520,6 +541,17 @@ def _get_attrs_simple(self):
     def _get_attrs_complex_list(self):
         return []
 
+    @staticmethod
+    def merge(columns):
```

Comment:
All of the merges should work on 1-length lists anyway, so you could eliminate the if statements and just do the extra work.